common/src/main/java/dev/lavalink/youtube/clients/ClientConfig.java

```diff
@@ -173,7 +173,8 @@ public String toJsonString() {
     public enum AndroidVersion {
         // https://apilevels.com/
         ANDROID_13("13", 33),
-        ANDROID_12("12", 31), // 12L => 32
+        ANDROID_12("12", 31),
+        ANDROID_12L("12L", 32),
```

Comment:
Bump this above `ANDROID_12` but below `ANDROID_13`.
Ordering is based on SDK version, descending.